phpstan has validation error when re-running checks with current version

PHPStan is showing the error below.
```
------ -------------------------------------------------------------------
  Line   Util/TSignalsDispatcher.php
 ------ -------------------------------------------------------------------
  238    Callable callable(): mixed invoked with 2 parameters, 0 required.
         🪪  arguments.count
 ------ -------------------------------------------------------------------

 [ERROR] Found 1 error
```
It may be from a phpstan version upgrade.

I originally wrote that, but I also have no idea what the issue means or how to fix it.

--- a/framework/Util/TSignalsDispatcher.php
+++ b/framework/Util/TSignalsDispatcher.php
@@ -235,7 +235,9 @@ class TSignalsDispatcher extends TComponent implements \Prado\ISingleton
 			$callable = is_callable($handler);
 			if ($callable) {
 				$handler = function ($sender, $param) use ($handler) {
-					return $handler($param->getSignal(), $param->getParameter());
+					if (is_callable($handler)) {
+						return $handler($param->getSignal(), $param->getParameter());
+					}
 				};
 				self::$_priorHandlers[$signal][1] = $handler;
 			}

